ironrdp-cli/src/gui/input.rs

```diff
@@ -0,0 +1,99 @@
+use std::sync::mpsc::Receiver;

```

Comment:
This module appears to be unused now in `ironrdp-cli`. Do you think you could remove it?

Reply:
i checked out master for the ironrdp-cli folder and didnt realize it would not remove existing file. Fixed